packages/shared-ux/chrome/navigation/src/ui/components/navigation_item.tsx

```diff
@@ -19,7 +19,6 @@ export interface Props<
   Id extends string = string,
   ChildrenId extends string = Id
 > extends NodeProps<LinkId, Id, ChildrenId> {
-  element?: string;
```

Comment:
I removed this prop as it is no needed. For custom UI consumers should use the `unstyled` prop.